### DISCUSSION TITLE
Support Matrix additional to Discord

### DISCUSSION THREAD

**Discussion by JokerGermany** [rustdesk/rustdesk#696] [General]
Would be great if you would let the user the choice if the want to use a free or proprietary chat protocol and support Matrix (element.io), too.

Would be cool if you you would bridge your discord Server with Matrix?
https://t2bot.io/discord/

**Comment by G2G2G2G** [comment]
where is their use of discord, just the link to where they chat? They can use botnet chat all they want, not like many of us want to chat with them anyway
